app/app.go

```diff
@@ -105,6 +105,7 @@ var (
 		kavadist.ModuleName:         {supply.Minter},
 		issuance.ModuleAccountName:  {supply.Minter, supply.Burner},
 		hard.ModuleAccountName:      {supply.Minter},
+		swap.ModuleAccountName:      nil,
```

Comment:
Just want to note that if we support LP tokens in a future upgrade, we'll have to update this to Minter + Burner here AND write a migration for it (simply updating permissions in app.go doesn't propagate during an upgrade).

Reply:
👍 good reminder, I forgot that migration is required once update

Reply:
added code comment for next PR for reminder